packages/main/src/pages/protected/choose-workspace/ChooseWorkspace.jsx

```diff
@@ -35,9 +35,20 @@ export default function Index() {
     }
   }, []);
   const history = useHistory();
+  //text align right for arabic and hebrew
```

Comment:
i'm pretty sure this function is already implemented

Reply:
yes i've removed it

Reply:
You've not removed this textAlign function @akarasauce

Reply:
please re-review my pull request 🥺@JulianaSau